pxtlib/browserutils.ts

```diff
@@ -470,15 +470,19 @@ namespace pxt.BrowserUtils {
     }
 
     export function scaleImageData(img: ImageData, scale: number): ImageData {
-        const cvs = document.createElement("canvas");
-        cvs.width = img.width * scale;
-        cvs.height = img.height * scale;
-        const ctx = cvs.getContext("2d")
+        const inputCanvas = document.createElement("canvas");
+        const outputCanvas = document.createElement("canvas");
+        inputCanvas.width = img.width;
```

Comment:
Do the input canvas dimensions need to be multiplied by scale in the same way the output canvas dimensions are?

Reply:
No, they don't. See the same process below in `imageDataToPNG`.